pkg/tests/apis/secret/keeper_test.go

```diff
@@ -222,4 +222,122 @@ func TestIntegrationKeeper(t *testing.T) {
 			require.Empty(t, rawList.Items)
 		})
 	})
+
+	t.Run("creating keepers in multiple namespaces", func(t *testing.T) {
+		adminOrg1 := helper.Org1.Admin
+		adminOrgB := helper.OrgB.Admin
+
+		keeperOrg1 := mustGenerateKeeper(t, helper, adminOrg1, nil)
+		keeperOrgB := mustGenerateKeeper(t, helper, adminOrgB, nil)
```

Comment:
Did you mean to name the variables `keeperOrg1` and `keeperOrg2` or `keeperOrgA` and keeperOrgB`?

Reply:
Let's go with Org 1 & Org 2 for now, I opened a PR to change the underlying test helper that uses Org1 and OrgB: https://github.com/grafana/grafana/pull/98931